certs.go

```diff
@@ -0,0 +1,25 @@
+package empire
+
+import "golang.org/x/net/context"
+
+type certsService struct {
+	*Empire
+}
+
+func (s *certsService) CertsAttach(ctx context.Context, app *App, cert string) error {
+	app.Cert = cert
+
+	if err := s.store.AppsUpdate(app); err != nil {
+		return err
+	}
+
+	if err := s.releaser.ReleaseApp(ctx, app); err != nil {
```

Comment:
I'll follow up with this in a separate PR that makes the API simpler for creating new releases for apps.